pkg/brownfield/pathmaps_test.go

```diff
@@ -0,0 +1,58 @@
+// -------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+package brownfield
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ptv1 "github.com/Azure/application-gateway-kubernetes-ingress/pkg/apis/azureingressprohibitedtarget/v1"
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/tests/fixtures"
+)
+
+var _ = Describe("Test blacklisting path maps", func() {
+
+	appGw := fixtures.GetAppGateway()
+
+	pathMap1 := (*appGw.URLPathMaps)[0]
+	pathMap2 := (*appGw.URLPathMaps)[1]
+	pathMap3 := (*appGw.URLPathMaps)[2]
+
+	Context("Test GetBlacklistedHTTPSettings() with a blacklist", func() {
+		It("should create a list of blacklisted and non blacklisted path maps", func() {
+			// Blacklisted targets: Host: "bye.com", Paths: [/fox, /bar]
```

Comment:
does this comment needs correction ?

Reply:
It's actually correct - but I'm going to delete it - looking up what's blacklisted is a click away - into the definition of `GetAzureIngressProhibitedTargets()`